language/move-prover/src/bytecode_translator.rs

```diff
@@ -444,8 +444,7 @@ impl<'env> ModuleTranslator<'env> {
     /// Emit code for the function specification.
     fn generate_function_spec(&self, func_target: &FunctionTarget<'_>) {
         emitln!(self.writer);
-        SpecTranslator::new(self.writer, &func_target.func_env.module_env, true)
-            .translate_conditions(func_target);
+        SpecTranslator::new_for_spec_in_impl(self.writer, func_target, true).translate_conditions();
```

Comment:
Little confused about this one. Isn't this not for spec_in_impl?

Reply:
We are passing the same information as before.  It is just that the func_target is now being passed in the constructor and module_env is calculated from it internally inside SpecTranslator.